docs/zh/samples/serverless.md

```diff
@@ -0,0 +1,235 @@
+# 示例 - 如何运行在以Knative为例的Serverless环境中
+
+本示例以开源框架Knative为例子，演示如何在Serverless环境中通过Fluid进行统一的数据加速，本例子以AlluxioRuntime为例，实际上Fluid支持所有的Runtime运行在Serverless环境。
+
+
+## 安装
+
+1.根据[Knative文档](https://knative.dev/docs/install/serving/install-serving-with-yaml/)安装Knative Serving v1.2，需要开启[kubernetes.Deploymentspec-persistent-volume-claim](https://github.com/knative/serving/blob/main/config/core/configmaps/features.yaml#L156)和[kubernetes.podspec-persistent-volume-write](https://github.com/knative/serving/blob/main/config/core/configmaps/features.yaml#L161)。
+
+检查 Knative的组件是否正常运行
+
+```
+kubectl get Deployments -n knative-serving
+```
+
+> 注：本文只是作为演示目的，关于Knative的生产系统安装请参考Knative文档最佳实践进行部署。另外由于Knative的容器镜像都在gcr.io镜像仓库，请确保镜像可达。
+如果您使用的是阿里云，您也可以直接使用[阿里云ACK的托管服务](https://help.aliyun.com/document_detail/121508.html)降低配置Knative的复杂度。
+
+2.下载、安装Fluid最新版
+
+```
+git clone https://github.com/fluid-cloudnative/fluid.git
```

Comment:
I think it's better to use the chart repo for the chart installation in the documentation. Such as:

```
helm repo add fluid https://fluid-cloudnative.github.io/fluid
helm repo update
helm install fluid fluid/fluid xxxx
```

Reply:
Thanks. I will refer all the installation parts to one place.